projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -579,7 +591,10 @@ protected function render_response_key( $key, &$response, $is_user_logged_in ) {
 				$response[ $key ] = $this->site->get_p2_thumbnail_elements();
 				break;
 			case 'was_ecommerce_trial':
-				$response[ $key ] = $this->site->was_ecommerce_trial();
+				$response[ $key ] = $this->site->was_trial( self::$jetpack_enabled_trials[0] );
```

Comment:
Small nit: This `[0]` `[1]` indexing looks a bit odd to me, and what if folks add the third trial in between in the list?

Can we make `$jetpack_enabled_trials` a proper dictionary with keys being constants and check for that?

Reply:
Ok, sure.